includes/class-fs-api.php

```diff
@@ -462,7 +477,7 @@ private function toggle_force_http( $is_http ) {
 
             if ( $is_http ) {
                 Freemius_Api_WordPress::SetHttp();
-            } else {
+            } else if ( method_exists( 'Freemius_Api_WordPress', 'SetHttps' ) ) {

```

Comment:
Skipping HTTPS as a temp fix is the desired patch ?
For RemoteRequest, you did execute the same logic without calling the new function, here, SetHttps won't be called and that's it. By purpose ?